common/utils/util.js

```diff
@@ -10,4 +10,15 @@ const countWords = str => {
     .split(/\s+/).length
 }
 
-module.exports = { isEmptyObject, countWords }
+const removeUrlLevels = (url, levels) => {
+  if (!levels) {
+    return url
+  }
+  let newUrl = url
+  for (let i = 0; i < levels; i += 1) {
+    newUrl = newUrl.substring(0, newUrl.lastIndexOf('/'))
+  }
+  return newUrl
```

Comment:
I think I'd do this as a one liner 🤔 
`!levels ? url : url.split('/').slice(0,-levels).join('/')`